File: 1189-maximum-number-of-balloons/1189-maximum-number-of-balloons.cpp
class Solution {
public:
    int maxNumberOfBalloons(string text) {
        vector<int> count;
        int countB =0;
        int countA =0;
        int countL =0;
        int countO =0;
        int countN =0;
        for(int i =0 ; i<text.length() ; i++){
            if(text[i] == 'b'){
                countB++;
                
            }
            if(text[i] == 'a'){
                countA++;
                
            }
            if(text[i] == 'l'){
                countL++;
                
            }
            if(text[i] == 'o'){
                countO++;
            
            }
            if(text[i] == 'n'){
                countN++;
                
            }
        }
        return min({countB,countA,(countL/2),(countO/2),countN});
    }
};

File: 1189-maximum-number-of-balloons/1189-maximum-number-of-balloons.kt
class Solution {
    fun maxNumberOfBalloons(text: String): Int {
        var countB = 0;
        var countA = 0;
        var countL =0;
        var countO = 0;
        var countN = 0;
        for (i in text){
            if (i == 'b'){
                countB++;
            }
            if (i == 'a'){
                countA++;
            }
            if(i == 'l'){
                countL++;
                
            }
            if(i == 'o'){
                countO++;
            
            }
            if(i == 'n'){
                countN++;
                
            }
        }
        
        return minOf(countB,countA,(countL/2),(countO/2),countN)
    }
}